src/components/SkipNavigationLink/SkipNavigationLink.js

```diff
@@ -0,0 +1,13 @@
+import styles from './SkipNavigationLink.module.scss';
+import * as SELECTOR from 'constants/selectors';
```

Comment:
Nit pick: Would `SELECTORS` (plural) read more clearly? Or perhaps `import {MAIN_CONTENT_ID} from ...` directly?

Reply:
@apmatthews I would be in favor of updating to SELECTORS.

I feel `SELECTORS.MAIN_CONTENT_ID` reads better when there are many constants on a page and it helps visually namespace it, however seeing that there is only 1 right now I have no preference :)

Reply:
Works for me! I don't have a preference either but your reasoning for `SELECTORS` makes sense 👍🏻